<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dividergb_class</class>
 <widget class="QWidget" name="dividergb_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1610</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Divis√£o em RGB</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <widget class="QLabel" name="imgL1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <widget class="QLabel" name="imgL2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>20</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <widget class="QLabel" name="imgL3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>512</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
